projects/aca-playwright-shared/src/page-objects/components/aca-header.component.ts

```diff
@@ -28,6 +28,7 @@
  export class AcaHeader extends BaseComponent {
    private static rootElement = 'adf-toolbar';
    public createButton = this.getChild('[id="app.toolbar.create"]');
+   public viewDetails = this.getChild('[title="View Details"]');
```

Comment:
this button should already have an id, like the "createButton"

Reply:
can't find any 
![Screenshot 2023-07-11 at 07 42 53](https://github.com/Alfresco/alfresco-content-app/assets/122363592/986aa06f-ec61-48b1-b1e9-e31de49ccae2)

Reply:
this is not an ACA implementation but ADF is blocked by CF at the moment